ProjectSettings/ProjectSettings.asset

```diff
@@ -298,9 +298,8 @@ PlayerSettings:
     m_Enabled: 0
     m_Devices: []
   - m_BuildTarget: Standalone
-    m_Enabled: 1
-    m_Devices:
-    - Oculus
+    m_Enabled: 0
```

Comment:
was there a reason why oculus was removed?

Reply:
Not sure why either, I didn't personally change this